search-solr/src/main/java/edu/unc/lib/boxc/search/solr/services/SearchStateFactory.java

```diff
@@ -405,14 +405,9 @@ private void populateSearchState(SearchState searchState, Map<String,String[]> r
                 parameter = getParameter(request, searchSettings.searchFieldParam(
                         SearchFieldKey.DATE_CREATED_YEAR.name()));
                 if (parameter != null && parameter.length() > 0) {
-                    String[] dateRange = parameter.split(",");
-                    dateCreatedYear.setLeftHand(dateRange[0]);
-                    dateCreatedYear.setRightHand(dateRange[1]);
-                }
-
-                if (dateCreatedYear.getLeftHand() != null && dateCreatedYear.getRightHand() != null) {
-                    searchState.getRangeFields().put(SearchFieldKey.DATE_CREATED_YEAR.name(), dateCreatedYear);
+                    new SearchState.RangePair(parameter);
                 }
+                searchState.getRangeFields().put(SearchFieldKey.DATE_CREATED_YEAR.name(), dateCreatedYear);
```

Comment:
Should move this line up inside of the `if` starting at line 407 so that we aren't adding DATE_CREATED_YEAR when not actually requested.